aztk/node_scripts/install/node_scheduling.py

```diff
@@ -0,0 +1,60 @@
+from azure import batch
+from aztk.models import ClusterConfiguration, SchedulingTarget
+from core import config, log
+
+def disable_scheduling(batch_client: batch.BatchServiceClient):
+    """
+    Disable scheduling for the current node
+    """
+    pool_id = config.pool_id
+    node_id = config.node_id
+
+    node = batch_client.compute_node.get(pool_id, node_id)
+    if node.scheduling_state == batch.models.SchedulingState.enabled:
+        batch_client.compute_node.disable_scheduling(pool_id, node_id)
+        log.info("Disabled task scheduling for this node")
+    else:
+        log.info("Task scheduling is already disabled for this node")
+
+def enable_scheduling(batch_client: batch.BatchServiceClient):
+    """
+    Disable scheduling for the current node
+    """
+    pool_id = config.pool_id
+    node_id = config.node_id
+
+    node = batch_client.compute_node.get(pool_id, node_id)
+    if node.scheduling_state == batch.models.SchedulingState.disabled:
+        batch_client.compute_node.disable_scheduling(pool_id, node_id)
+        log.info("Enabled task scheduling for this node")
+    else:
+        log.info("Task scheduling is already enabled for this node")
+
+
+def setup_node_scheduling(
```

Comment:
Is this backwards compatible? Or is this breaking for existing clusters?

Reply:
This shouldn't i guess, the actual scheduling shouldn't change